api/src/services/FedoraCatalog.ts

```diff
@@ -0,0 +1,51 @@
+import Config from "../models/Config";
+
+export interface FedoraDatastream {
+    mimetype?: {
+        allowedType: string;
```

Comment:
I notice that allowedType is singular but allowedSubtypes is plural. It would probably make more sense to make them consistent, since they both work the same way -- I would vote for changing allowedType to allowedTypes.

Reply:
So, per allowed type there are multiple subtypes. That's why I kept it singular.

Reply:
True. I guess I was just thinking of the case where allowedType is a wildcard, in which case multiple values will match... but until there's a use case for a comma-separated list of types (which so far there is not), this is probably fine. :-)